src/components/Header_type_1.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
```

Comment:
Can we come up with a more descriptive name for the header, like `HeaderUnauthenticated` or `HeaderPreAuthenticated`